WooCommerce/Classes/AppDelegate.swift

```diff
@@ -249,6 +249,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 }
 
+// MARK: - Helper method for WooCommerce POS
+//
+extension AppDelegate {
+    func shouldHideTabBar(_ hidden: Bool) {
```

Comment:
maybe call it `setShouldHideTabBar` since it is **setting** and not **getting**?

Reply:
Updated on https://github.com/woocommerce/woocommerce-ios/pull/12700/commits/5a073b03a967561e1aa8a34bc7df1f549c3b7efa but I don't like neither tbh 😂 , maybe somebody else has a better name for it!